docs/src/components/header/HeaderNavBar.tsx

```diff
@@ -317,6 +317,11 @@ export default function HeaderNavBar() {
             About us
           </Link>
         </li>
+        <li role="none">
+          <Link role="menuitem" href={ROUTES.newCareers}>
+            Careers
```

Comment:
We can remove this one from the HeaderNavBar. It's enough having it on the footer :)